integrationtest/unregisterclient/main.d

```diff
@@ -0,0 +1,123 @@
+/*******************************************************************************
+
+    Test for EpollSelectDispatcher.unregister behaviour
+
+    Copyright:
+        Copyright (c) 2018 sociomantic labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module integrationtest.unregisterclient.main;
+
+import ocean.transition;
+
+import ocean.io.device.File;
+import ocean.io.select.EpollSelectDispatcher;
+import ocean.io.select.client.SelectEvent;
+import ocean.util.app.DaemonApp;
+import ocean.util.test.DirectorySandbox;
+import ocean.core.Test;
+
+/// Main application class
+class EpollTestApp : DaemonApp
```

Comment:
Why is this a daemon app btw? You don't seem to be using any of its facilities.

Reply:
It's just the easiest way for me to write these tests - just copy the boilerplate from another test. I'll make it non-daemon app.

Reply:
I mean, do you need an app class at all?

Reply:
No, that's what I meant by making it non-daemon app - I'll remove all classes.